source/Meadow.Core/Meadow.Core.csproj

```diff
@@ -21,8 +21,8 @@
 	<ItemGroup>
 		<None Include="..\icon.png" Link="icon.png" Pack="true" PackagePath="" />
     <ProjectReference Include="..\..\..\Meadow.Contracts\Source\Meadow.Contracts\Meadow.Contracts.csproj" />
-		<PackageReference Include="Meadow.Modbus" Version="0.98.0" />
```

Comment:
Is swapping package with project references intentional here?

Reply:
for `develop` we shouldn't be using nugets, only project refs